static/css/restyle/restyle.less

```diff
@@ -2050,3 +2050,51 @@ body.editor-tools {
   }
 
 }
+
+/* Source/Diff viewer */
+body.file-viewer {
+  pre, code, kbd, tt, samp, tt {
+    font-family: "andale mono", monospace !important;
```

Comment:
Ah yes, I did get it backwards regarding specificity, probably because i missed the `!important` in the `*` rule. But it has the same end result, with the `*` rule using `!important` overriding any other font-family definitions, we need to use `!important` for any font-family rule in the future. If we could get rid of this hack at some point that would be great, e.g. by cleaning up existing font-family rules. In the company I used to work for we even had a linter rule that !important can only be used followed by a comment why it is really needed :)